scripts/mixins/families/chigoe.lua

```diff
@@ -5,6 +5,20 @@ require('scripts/globals/mixins')
 g_mixins = g_mixins or {}
 g_mixins.families = g_mixins.families or {}
 
+local jobAbilities = set({
+    xi.jobAbility.SHIELD_BASH,
+    xi.jobAbility.JUMP,
+    xi.jobAbility.HIGH_JUMP,
+    xi.jobAbility.WEAPON_BASH,
+    xi.jobAbility.CHI_BLAST,
+    xi.jobAbility.TOMAHAWK,
+    xi.jobAbility.ANGON,
+    xi.jobAbility.QUICKSTEP,
+    xi.jobAbility.BOXSTEP,
+    xi.jobAbility.STUTTER_STEP,
+    xi.jobAbility.FEATHER_STEP,
+})
```

Comment:
I dont think the parenthesis after `set` is needed here.
```lua
local jobAbilities =
set{
    xi.thing
}
```

should work just fine

Reply:
updated